clang/test/AST/HLSL/ByteAddressBuffer-AST.hlsl

```diff
@@ -0,0 +1,49 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.0-library -x hlsl -ast-dump -DEMPTY \
```

Comment:
Since the test now covers multiple buffers please rename it to ByteAddressBuffer**s**-AST.hlsl and update the PR title and description.